### ISSUE TITLE
Warnings in Protégé

### ISSUE THREAD

**Issue by carueda** [ESIPFed/sweet#57]
@lewismc @dr-shorthair 

I just loaded http://sweetontology.net/sweetAll in Protégé 5.2.0.

This was initially as an extra verification of the COR-enabled resolution (#51), which seems to be properly in place.

However, upon looking at the log ([protege.log](https://github.com/ESIPFed/sweet/files/1415570/protege.log) -- I have manually removed some of the most irrelevant lines) I note several WARNs associated with SWEET.  Most of them seem to be Annotation property warnings, for example:

```
Annotation property range axiom turned to object property range after parsing. 
  This could introduce errors if the original range was an anonymous expression: 
  http://sweetontology.net/propTime/Period is the new domain.

Annotation property range axiom turned to object property range after parsing. 
  This could introduce errors if the original range was an anonymous expression: 
  http://sweetontology.net/propTimeFrequency/Frequency is the new domain.

Annotation property range axiom turned to object property range after parsing. 
  This could introduce errors if the original range was an anonymous expression: 
  http://sweetontology.net/propTime/Time is the new domain.
```

You are probably already aware of this. Feel free to proceed as you deem appropriate.

**Comment by carueda** [comment]
Copying note just added to the referenced bioportal issue:

>Here's the discussion on Protege's user support mailing list: http://protege-project.136.n4.nabble.com/Debugging-owlapi-errors-on-SWEET-ontology-td4664108.html.

**Comment by lewismc** [comment]
I just loaded SWEET master into Protege 5.0.0 and there are many more issues. I've attached my log file such that we can work our way through it.
As you can see @carueda there are possibly hundreds of instances where **unparsed triple** is present.
[protege.log](https://github.com/ESIPFed/sweet/files/1596128/protege.log)

**Comment by dr-shorthair** [comment]
Error at line 111 - could this cascade to later problems? 
18:57:55.110 [Installing plugins thread] ERROR PluginInstaller    An error occurred whilst downloading and installing the OWLAPI RDF Library plugin: https://github.com/protegeproject/rdf-library/releases/download/rdf-library-2.0.2/rdf-library-2.0.2.jar

**Comment by lewismc** [comment]
Very possibly @dr-shorthair I'll scope this out closer to ESIP. Hopefully we can get a few fixes in place and possibly even make a 3.2.0 release.

**Comment by lewismc** [comment]
Superseded by #124
